deploy/buildtools/buildfarm.py

```diff
@@ -133,7 +123,7 @@ def release_build_host(self, build_config: BuildConfig) -> None:
         return
 
 
-class IPAddrBuildFarm(BuildFarm):
+class UnmanagedBuildFarm(BuildFarm):
```

Comment:
Ok so since we have eyes on this, we should probably decide on a name. (I don't think this should block merge though). 

Maybe "UserManaged" or "UserDefined" is better as they better echo the comment below.